spec/lib/embed/viewer/geo_spec.rb

```diff
@@ -50,13 +50,13 @@
     end
     it 'generates a file list when file has resources' do
       stub_purl_response_and_request(geo_purl_public, request)
-      html = Capybara.string(geo_viewer.download_html)
+      html = Capybara.string(geo_viewer.download_html.to_html)
```

Comment:
Yeah, I struggled with this interface a bit.  Here is what I was trying to accomplish with this, and let's see if we can come up with a more appropriate alternative.

The `download_html` method **used** to return a string (which happened to be the result of the 'to_html' method on the `Nokogiri::HTML::Builder` object).  This method now returns an instance of `Embed::DownloadPanel`, which is why we are calling `to_html` on the resulting object.

`Embed::DownloadPanel`'s responsibility is to return HTML which is why it has a `to_html` method.  The **actual** implementation is more of a string, so the class delegates `to_str` to `to_html` (which allows the `... << download_html << ...` to work in [`CommonViewer`](https://github.com/sul-dlss/sul-embed/blob/master/lib/embed/viewer/common_viewer.rb#L41)).

I can think of a few immediate options.

* Require the viewers implementing `Embed::DownloadPanel` to explicitly call `to_html` (which would mimic how we're using `Nokogiri::HTML::Builder`)
* Move everything into string methods like `to_s` (I feel like I went down this route at one point, but am willing to explore again).